src/utils/sessionXCSpy.ts

```diff
@@ -21,10 +21,17 @@ class SessionXCSpy {
             });
         });
         xcomponentapi.setLogLevel(xcLogLevel.INFO);
+    }
+
+    getDefaultPrivateTopic(): string {
```

Comment:
Prefer the use of get / set typescript definition
http://www.codingdefined.com/2015/05/get-and-set-property-in-typescript.html